README.md

```diff
@@ -416,321 +412,20 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
 
 ---
 
-## Selection of Compilers
```

Comment:
Maybe I missed it, but I did not see the links to these new .rst files in README.

Reply:
The readme links to the documentation as a unit now, and the documentation now has a Get Started section with "Selecting a Compiler" and all the build instructions.